### PR TITLE
31 fix h1 headings

### PR DESCRIPTION
**Pull request by gtdang**
Fixed headings.

### CODE REVIEW

**Review by listar2000 — APPROVED** [review]
Looks good to me!

**Review by benwandrew — APPROVED** [review]
terrific!

**Review by chadcwilliams — APPROVED** [review]
Looks good to me!

File: git-text-content.md [hunk 1]
```diff
@@ -319,7 +319,7 @@ References[edit]
 - [^121] "Why does Git use a cryptographic hash function?". Stack Overflow. 1 March 2015. Archived from the original on 1 July 2016.
 - [^122]: "Git – hash-function-transition Documentation". git-scm.com.
 
-External links[edit]
+# External links[edit]
```
Comment (younesStrittmatter) [inline git-text-content.md:322]: this looks great!

**Review by younesStrittmatter — APPROVED** [review]
Well done! Ready to merge

**Review by mikakoinaba — COMMENTED** [review]
Everything looks good!

**Review by TheLemonPig — APPROVED** [review]
Looks good!

File: git-text-content.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ Stupid. Contemptible and despicable. Simple. Take your pick from the dictionary
 "Goddamn idiotic truckload of sh*t": when it breaks.
 The source code for Git refers to the program as, "the information manager from hell."
 
-Design
+# Design
```
Comment (musslick) [inline git-text-content.md:34]: Looks great!

File: git-text-content.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ Git (/ɡɪt/)[8] is free and open source software for distributed version contro
 
 Git was originally authored by Linus Torvalds in 2005 for development of the Linux kernel, with other kernel developers contributing to its initial development.[12] Since 2005, Junio Hamano has been the core maintainer. As with most other distributed version control systems, and unlike most client–server systems, every Git directory on every computer is a full-fledged repository with complete history and full version-tracking abilities, independent of network access or a central server.[13] Git is free and open-source software distributed under the GPL-2.0-only license.
 
-## History
+# History
```
Comment (musslick) [inline git-text-content.md:9]: Looking good...

File: git-text-content.md [hunk 4]
```diff
@@ -70,7 +70,7 @@ When there are more than one common ancestors that can be used for a three-way m
 
 — Linus Torvalds[61]
 octopus: This is the default when merging more than two heads.
-Data structures[edit]
+## Data structures[edit]
```
Comment (musslick) [inline git-text-content.md:73]: Not sure if this should be a heading? Shouldn't this be a list?

File: git-text-content.md (hunk 1 above)
Comment (musslick) [inline git-text-content.md:322]: Looks good!

**Review by musslick — COMMENTED** [review]
I'm not sure about one headline (see comment) which could be a list element

**Pull request merged**